MapboxMobileEventsTests/MMEEventTests.mm

```diff
@@ -63,6 +65,25 @@ + (NSDictionary *)nilAttributes;
             unarchived should_not be_nil;
             unarchived should equal(event);
         });
+        
+        
+        it(@"should delete stored event when encoded version is greater than MMEEventVersion", ^{
```

Comment:
this test description isn't quite correct, the event is encoded into memory and we validate that it decodes to nil.

we need a test to cover writing that data to the `pendingMetricsEventPath` then verify that `loadPendingTelemetryMetricsEvent` returns nil and that no file exists at the pending event path

Reply:
Thanks for pointing that out. I've added the test and corrected the description.